charts/hdfs-namenode-k8s/templates/namenode-statefulset.yaml

```diff
@@ -60,13 +87,13 @@ spec:
               value: "true"
             - name: CORE_CONF_hadoop_rpc_protection
               value: privacy
-            - name:   HDFS_CONF_dfs_block_access_token_enable
+            - name: HDFS_CONF_dfs_block_access_token_enable
               value: "true"
-            - name:   HDFS_CONF_dfs_encrypt_data_transfer
+            - name: HDFS_CONF_dfs_encrypt_data_transfer
               value: "true"
             - name: HDFS_CONF_dfs_namenode_kerberos_principal
               value: hdfs/_HOST@{{ required "A valid kerberosRealm entry required!" .Values.kerberosRealm }}
-            - name:  HDFS_CONF_dfs_namenode_kerberos_https_principal
+            - name: HDFS_CONF_dfs_namenode_kerberos_https_principal
```

Comment:
from the other configs it seems we only support http access...assuming https is just hadoop weirdness in naming... 

Also, please check this property.. I am not seeing this is [new hadoop docs](https://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-common/SecureMode.html) .. Saw it in the [older docs](https://hadoop.apache.org/docs/r2.4.1/hadoop-project-dist/hadoop-common/SecureMode.html).